template.yaml

```diff
@@ -659,6 +659,27 @@ Resources:
             Method: post
             RestApiId: !Ref NvaPublicationApi
 
+  UpdateTicketHandler:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: tickets
+      Handler: no.unit.nva.publication.ticket.update.UpdateTicketHandler::handleRequest
+      Runtime: java11
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          TABLE_NAME: !Ref NvaResourcesTable
+          BACKEND_CLIENT_AUTH_URL: !Ref CognitoAuthorizationUri
+          BACKEND_CLIENT_SECRET_NAME: 'BackendCognitoClientCredentials'
+      Role: !GetAtt LambdaRole.Arn
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Path: /{publicationIdentifier}/ticket/{ticketIdentifier}/update
```

Comment:
Hvorfor `/update` på slutten? PUT gjøres typisk på ressurs-url'en. `/{publicationIdentifier}/ticket/{ticketIdentifier}`

Reply:
Because this endpoint is in use (of function that should be removed after we deploy new one), I can remove this old function, but frontend should take in use new handler pretty fast, so ticket flow works properly. 

  UpdateTicketFunction:
    Type: AWS::Serverless::Function
    Properties:
      CodeUri: tickets
      Handler: no.unit.nva.publication.ticket.update.UpdateTicketStatusHandler::handleRequest
      Runtime: java11
      Timeout: 30
      Environment:
        Variables:
          ALLOWED_ORIGIN: '*'
          TABLE_NAME: !Ref NvaResourcesTable
          API_HOST: !Ref ApiDomain
          BACKEND_CLIENT_AUTH_URL: !Ref CognitoAuthorizationUri
          BACKEND_CLIENT_SECRET_NAME: 'BackendCognitoClientCredentials'
      Role: !GetAtt LambdaRole.Arn
      Events:
        GetEvent:
          Type: Api
          Properties:
            Path: /{publicationIdentifier}/ticket/{ticketIdentifier}
            Method: put
            RestApiId: !Ref NvaPublicationApi